src/constants/bucketNames.ts

```diff
@@ -10,3 +10,8 @@ export const TREND = "trend";
 export const SEGMENT = "segment";
 export const ROWS = "rows";
 export const COLUMNS = "columns";
+export const LOCATION = "location";
+export const SIZE = "size";
```

Comment:
why you dont use 'measures' and 'secondary_measures' internally? Sematics is given by prop names in UI SDK component and by Titles in AD buckets

Reply:
Do you mean `SIZE = "measures"` and `COLOR = "secondary_measures"`?
I think that it would help in case we create mock data with semantic values, like `localIdentifier: "size"` rather than `localIdentifier: "measures"`.

And not sure that I get you right.